test/core/proofs/proofs_test.cpp

```diff
@@ -191,7 +191,7 @@ TEST_F(ProofsTest, DISABLED_ValidPoSt) {
  * @when Commit and Verifies Seal and Unseal
  * @then success
  */
-TEST_F(ProofsTest, ValidSealAndUnseal) {
+TEST_F(ProofsTest, DISABLED_ValidSealAndUnseal) {
```

Comment:
Please add jira task id that describes the problem and possible solutions, which are local for CI cache or a cmake flag for enabling proofs_test target.

Reply:
Added